oam/install_scripts/post-install

```diff
@@ -234,31 +254,49 @@ if [ $user = "root" ]; then
 fi
 
 if [ $user = "root" ]; then
-	$installdir/bin/syslogSetup.sh install > /tmp/syslog_install.log 2>&1
-	rm -f /etc/default/columnstore
+	$installdir/bin/syslogSetup.sh install > $tmpDir/syslog_install.log 2>&1
+
+	#check if MariaDB Columnstore system logging was setup
+	cat $tmpDir/syslog_install.log | grep 'No System Logging' >/dev/null 2>&1
+	if [ $? -eq 0 ]; then
+	    cat $tmpDir/syslog_install.log
```

Comment:
I don't know what the syslog_install.log file looks like.  If it's verbose, it would be more clear to print that a logging system wasn't found and suggest an action.